src/Routes/PeopleList/index.js

```diff
@@ -0,0 +1,27 @@
+import { useSelector } from "react-redux";
+import { selectPeople } from "../../peopleSlice";
+import PersonCardToPeoplePage from "../../common/PersonCardToPeoplePage";
+import { usePeopleData } from "../../usePeopleData";
+import { Headline, MainArticle, StyledSection } from "./styled";
+
+function PeopleList() {
+  usePeopleData();
+  const people = useSelector(selectPeople);
+
+  return (
```

Comment:
render triggers even with a single object in the state causing an error when returning from peoplePage